Test failures on a fresh clone of dev

```
npm install
gulp test
```

Output:

```
  1) FileSink should format events as JSON:

      Uncaught AssertionError: false == true
      + expected - actual

      -false
      +true

      at C:\Development\Open Source\nblumhardt\structured-log\test\file-sink.tests.js:47:7
      at WriteStream.onEnd (C:\Development\Open Source\nblumhardt\structured-log\src\core\structured-log.js:242:9)
      at finishMaybe (_stream_writable.js:484:14)
      at afterWrite (_stream_writable.js:362:3)
      at onwrite (_stream_writable.js:352:7)
      at WritableState.onwrite (_stream_writable.js:105:5)
      at fs.js:1799:5
      at FSReqWrap.strWrapper (fs.js:568:5)

  2) LoggerConfiguration #minimumLevel() should suppress events below minimum level:

      AssertionError: 0 == 1
      + expected - actual

      -0
      +1

      at Context.<anonymous> (C:\Development\Open Source\nblumhardt\structured-log\test\structured-log.tests.js:14:14)

  3) LoggerConfiguration #minimumLevel() should apply in pipeline order:
     TypeError: undefined is not a function
      at Context.<anonymous> (C:\Development\Open Source\nblumhardt\structured-log\test\structured-log.tests.js:37:11)

  4) LoggerConfiguration #writeTo() should report failures:
     TypeError: undefined is not a function
      at Context.<anonymous> (C:\Development\Open Source\nblumhardt\structured-log\test\structured-log.tests.js:64:11)
```

Looking into it now.

--- a/test/file-sink.tests.js
+++ b/test/file-sink.tests.js
@@ -44,7 +44,7 @@ describe('FileSink', function() {
       assert(jvent.message === 'Hello, world!');
       assert(jvent.messageTemplate === 'Hello, {name}!');
       assert(jvent.properties.name === 'world');
-      assert(jvent.level === 'INFORMATION');
+      assert(jvent.level === 'INFO');
       done();
     });
   });

--- a/test/structured-log.tests.js
+++ b/test/structured-log.tests.js
@@ -6,7 +6,7 @@ describe('LoggerConfiguration', function() {
     it('should suppress events below minimum level', function() {
       var written = [];
       var log = serilog.configuration()
-        .minimumLevel('WARNING')
+        .minimumLevel('WARN')
         .writeTo(function(evt) { written.push(evt); })
         .createLogger();
       log('Today is sunnny and clear');
@@ -17,7 +17,7 @@ describe('LoggerConfiguration', function() {
     it('should permit events above minimum level', function() {
       var written = [];
       var log = serilog.configuration()
-        .minimumLevel('WARNING')
+        .minimumLevel('WARN')
         .writeTo(function(evt) { written.push(evt); })
         .createLogger();
       log.error('The sky is falling!');
@@ -29,12 +29,12 @@ describe('LoggerConfiguration', function() {
       var info = [];
       var errs = [];
       var log = serilog.configuration()
-        .minimumLevel('INFORMATION')
+        .minimumLevel('INFO')
         .writeTo(function(evt) { info.push(evt); })
         .minimumLevel('ERROR')
         .writeTo(function(evt) { errs.push(evt); })
         .createLogger();
-      log.warning('Today is stormy');
+      log.warn('Today is stormy');
 
       assert.equal(1, info.length);
       assert.equal(0, errs.length);
@@ -61,7 +61,7 @@ describe('LoggerConfiguration', function() {
         })
         .writeTo(function(evt) { written.push(evt); })
         .createLogger();
-      log.warning('A timely warning');
+      log.warn('A timely warning');
 
       assert.equal(2, written.length);
 
@@ -69,7 +69,7 @@ describe('LoggerConfiguration', function() {
       assert(written[0].properties.isSelfLog);
       assert.notEqual(-1, written[0].renderedMessage().indexOf('Broken!'));
 
-      assert.equal('WARNING', written[1].level);
+      assert.equal('WARN', written[1].level);
       assert(!written[1].properties.isSelfLog);
     });
   });
@@ -116,44 +116,44 @@ describe('LoggerConfiguration', function() {
 
 describe('serilog.event()', function(){
   it('should capture message template parameters', function(){
-    var evt = serilog.event('INFORMATION', 'Happy {age}th birthday, {name}!', 30, 'Fred');
+    var evt = serilog.event('INFO', 'Happy {age}th birthday, {name}!', 30, 'Fred');
     assert.equal(30, evt.properties.age);
     assert.equal('Fred', evt.properties.name);
   });
 
   it('should render messages', function(){
-    var evt = serilog.event('INFORMATION', 'Happy {age}th birthday, {name}!', 30, 'Fred');
+    var evt = serilog.event('INFO', 'Happy {age}th birthday, {name}!', 30, 'Fred');
     assert.equal('Happy 30th birthday, Fred!', evt.renderedMessage());
   });
 
   it('should capture extra parameters', function(){
-    var evt = serilog.event('INFORMATION', 'Hi, {name}!', 'Fred', 'Smith');
+    var evt = serilog.event('INFO', 'Hi, {name}!', 'Fred', 'Smith');
     assert.equal('Smith', evt.properties.a1);
     assert.equal('Fred', evt.properties.name);
   });
 
   it('should render missing parameters', function(){
-    var evt = serilog.event('INFORMATION', 'Happy {age}th birthday, {name}!', 30);
+    var evt = serilog.event('INFO', 'Happy {age}th birthday, {name}!', 30);
     assert.equal('Happy 30th birthday, {name}!', evt.renderedMessage());
   });
 
   it('should suppress missing properties', function(){
-    var evt = serilog.event('INFORMATION', 'Happy {age}th birthday, {name}!', 30);
+    var evt = serilog.event('INFO', 'Happy {age}th birthday, {name}!', 30);
     assert(!evt.properties.hasOwnProperty('name'));
   });
 
   it('should survive duplicate properties', function(){
-    var evt = serilog.event('INFORMATION', 'Hi, {name} {name}!', 'Fred', 'Smith');
+    var evt = serilog.event('INFO', 'Hi, {name} {name}!', 'Fred', 'Smith');
     assert.equal('Smith', evt.properties.name);
   });
 
   it('should stringify objects', function(){
-    var evt = serilog.event('INFORMATION', 'Hi, {person}!', { name: 'Fred' });
+    var evt = serilog.event('INFO', 'Hi, {person}!', { name: 'Fred' });
     assert.equal('string', typeof evt.properties.person);
   });
 
   it('should observe destructuring hints', function(){
-    var evt = serilog.event('INFORMATION', 'Hi, {@person}!', { name: 'Fred' });
+    var evt = serilog.event('INFO', 'Hi, {@person}!', { name: 'Fred' });
     assert.equal('object', typeof evt.properties.person);
     assert.equal('Fred', evt.properties.person.name);
   });

